test/lib/ansible_test/_internal/coverage/combine.py

```diff
@@ -137,11 +141,16 @@ def _command_coverage_combine_powershell(args):
     """
     coverage_files = get_powershell_coverage_files()
 
-    def _default_stub_value(lines):
-        val = {}
-        for line in range(lines):
-            val[line] = 0
-        return val
+    if args.stub:
+        find_executable('pwsh', required=True)
+
+    def _default_stub_value(sources):
+        cmd = ['pwsh', os.path.join(ANSIBLE_TEST_DATA_ROOT, 'coverage_stub.ps1')]
+        cmd.extend(sources)
+
+        stubs = json.loads(run_command(args, cmd, capture=True, always=True)[0])
+
+        return dict((d['Path'], dict((line, 0) for line in d['Lines'])) for d in stubs)
```

Comment:
Rename `sources` to avoid PyCharm complaining about variable shadowing.

```suggestion
    def _default_stub_value(source_paths):
        cmd = ['pwsh', os.path.join(ANSIBLE_TEST_DATA_ROOT, 'coverage_stub.ps1')]
        cmd.extend(source_paths)

        stubs = json.loads(run_command(args, cmd, capture=True, always=True)[0])

        return dict((d['Path'], dict((line, 0) for line in d['Lines'])) for d in stubs)
```